examples/XmlExample.php

```diff
@@ -86,73 +78,27 @@ private function addAttributes($item, $itemData)
         }
     }
 
-    private function addBonuses($item, $itemData)
+    private function addBonuses(Item $item, $itemData)
     {
-        $bonusesData = [
-            '' => [
-                3
-            ],
-            'LNrLF7BRVJ0toQ==' => [
-                5
-            ]
-        ];
-
-        $bonusElement = new Bonus();
-
-        foreach ($bonusesData as $usergroup => $bonuses) {
-            foreach ($bonuses as $bonus) {
-                $bonusElement->setValue($bonus, $usergroup);
-            }
-        }
-
-        $item->setBonus($bonusElement);
+        $item->addBonus(3);
+        $item->addBonus(5, 'LNrLF7BRVJ0toQ==');
     }
 
-    private function addDateAddeds($item, $itemData)
+    private function addDateAddeds(Item $item, $itemData)
     {
-        $dateAddedsData = [
-            '' => [
-                new \DateTime()
-            ],
-            'LNrLF7BRVJ0toQ==' => [
-                new \DateTime()
-            ]
-        ];
-
-        $dateAddedElement = new DateAdded();
-
-        foreach ($dateAddedsData as $usergroup => $dateAddeds) {
-            foreach ($dateAddeds as $dateAdded) {
-                $dateAddedElement->setDateValue($dateAdded, $usergroup);
-            }
-        }
-
-        $item->setDateAdded($dateAddedElement);
+        $item->addDateAdded(new DateTime());
```

Comment:
Add the backslash for the global namespace or add a use statement.

Reply:
✅